geniza/corpus/sitemaps.py

```diff
@@ -0,0 +1,49 @@
+from datetime import date
+
+from django.contrib.sitemaps import Sitemap
+from django.urls import reverse
+
+from geniza.corpus.models import Document
+from geniza.corpus.solr_queryset import DocumentSolrQuerySet
+
+
+def solr_timestamp_to_date(timestamp):
+    """Convert solr isoformat date time string to python date."""
+    # format: 2020-05-12T15:46:20.341Z
+    # django sitemap only includes date, so strip off time
+    yearmonthday = timestamp.split("T")[0]
+    return date(*[int(val) for val in yearmonthday.split("-")])
+
+
+class DocumentSitemap(Sitemap):
+    def items(self):
+        return (
+            DocumentSolrQuerySet()
+            .filter(
+                status="Public"
+            )  # ?: It's not saved as "Document.PUBLIC" which is "P"
```

Comment:
Please add a constant to `Document` for this value — maybe `Document.PUBLIC_LABEL` ? and use it everywhere you have the string value now (status field choices, here, and search view, ...). 

Unless you think we should just index the P/S codes instead of the labels; in which case, you should update the index data and use the existing constant here. (I don't _think_ we're currently using it anywhere else, so this is probably fine — but if we ever wanted to use them for facets it wouldn't be very meaningful!)